packages/client/src/views/SysAdmin/Performance/Vsexports/VSExport.tsx

```diff
@@ -0,0 +1,209 @@
+import React from 'react'
+import { SysAdminContentWrapper } from '@client/views/SysAdmin/SysAdminContentWrapper'
+import { FormTabs } from '@opencrvs/components/lib/forms'
+import styled from 'styled-components'
+import { useIntl } from 'react-intl'
+import { BodyContent } from '@opencrvs/components/lib/layout'
+import { Content } from '@opencrvs/components/lib/interface/Content'
+import { messages } from '@client/i18n/messages/views/config'
+import {
+  ListViewItemSimplified,
+  ListViewSimplified
+} from '@opencrvs/components/lib/interface/ListViewSimplified/ListViewSimplified'
+import { Query } from '@client/components/Query'
+import { GET_TOTAL_VSEXPORT } from './queries'
+import {
+  Label,
+  Value
+} from '@client/views/SysAdmin/Config/Application/Components'
+import { LoadingIndicator } from '@client/views/OfficeHome/LoadingIndicator'
+
+import { MINIO_URL } from '@client/utils/constants'
+import { DynamicHeightLinkButton } from '@client/views/Settings/items/components'
+import { ErrorToastNotification } from '@opencrvs/components/lib/interface'
+
+import { buttonMessages, errorMessages } from '@client/i18n/messages'
+import { GetVsExportsQuery } from '@client/utils/gateway'
+
+const UserTable = styled(BodyContent)`
+  padding: 0px;
+  margin: 8px auto 0;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.md}px) {
+    padding: 0px;
+  }
+`
+type VsExportInterface = {
+  event: string
+  year: number
+  url: string
+  createdOn: string
+  fileSize: string
+}
```

Comment:
We won't need this anymore because we can reuse the type generated in the `gateway.ts` file